OpenRA.Game/Map/CellLayer.cs

```diff
@@ -155,7 +155,7 @@ public MPos Clamp(MPos uv)
 	// Helper functions
 	public static class CellLayer
 	{
-		/// <summary>Create a new layer by resizing another layer.  New cells are filled with defaultValue.</summary>
```

Comment:
`<summary>` is an IDE thing for code documentation and easy lookup. But I'm voting for keeping your change anyway.